steps/report.sh

```diff
@@ -57,6 +58,67 @@ fi
 
 sort -o "${list}" "${list}"
 
+# Create the aggregation table LaTeX file
+aggregation_table=${TARGET}/temp/aggregation_table.tex
+rm -f "${aggregation_table}"
+touch "${aggregation_table}"
+
+# LaTeX escape function to handle special characters
+latex_escape() {
+  echo "$1" | sed 's/&/\\&/g; s/%/\\%/g; s/_/\\_/g; s/\$/\\\$/g; s/#{}/\\{\\}/g; s/\^/\\^/g; s/~/{\~}/g; s/\\/\\\\/g'
+}
+
+{
+  printf "\onecolumn\n"
+  printf "\\centering\n"
+  printf "\\\\begin{longtable}{|l|c|c|c|}\n"
+  printf "\\hline\n"
+  printf "Metric & 90th Percentile & Mean & Median \\\\\\\\\\\\\ \n"
+  printf "\\hline\n"
+} >> "${aggregation_table}"
+
+files=("${TARGET}/data/aggregation/*.csv")
+
+# shellcheck disable=SC2128
+# I disabled it, because unwrapping array and take first elem is exactly what i need in this script
```

Comment:
@dzhovi can't you just do it the way shellcheck recommends?

Reply:
@yegor256 fixed